test/extensions/filters/common/expr/context_test.cc

```diff
@@ -205,7 +205,7 @@ TEST(Context, RequestFallbackAttributes) {
   Http::TestRequestHeaderMapImpl header_map{
       {":method", "POST"},
       {":scheme", "http"},
-      {":path", "/meow?yes=1"},
+      {":path", "/meow"},
```

Comment:
what changed about this test?

Reply:
There is an edge case of missing `?` that gets covered.